test_routes.py

```diff
@@ -0,0 +1,14 @@
+from test_client import client
+from unittest.mock import Mock, patch
+import trello_items
+import requests
+
+def mock_requests():
+    mock = Mock()
+    mock.configure_mock(return_value=True)
+    return mock
+
+@patch.object(requests, 'get', side_effect=mock_requests)
+def test_index_page(requests_mock, client):
```

Comment:
I can't get this test to work. No matter what I try, I can't mock the requests.get method in trello_items module.